libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1269,6 +1269,7 @@ class MapboxNavigation @VisibleForTesting internal constructor(
      * @see [ViewUtils.capture] to capture screenshots
      * @see [FeedbackHelper.encodeScreenshot] to encode screenshots
      */
+    @ExperimentalPreviewMapboxNavigationAPI
```

Comment:
Apparently now annotations are transitively enforced, would that be a breaking change? 🤔

Reply:
should help I think, that's because of FeedbackMetadata, which is `null` here
```suggestion
    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
```